core/model/src/main/java/org/eclipse/sw360/antenna/model/artifact/facts/javaScript/JavaScriptCoordinates.java

```diff
@@ -67,6 +68,11 @@ public String getType() {
         return "npm";
     }
 
+    @Override
+    protected PackageURLBuilder addPurlFacts(PackageURLBuilder builder) {
+        return builder.withNamespace(getName()).withName(getArtifactId());
```

Comment:
what about the `version`? It is handled somewhere else?

Reply:
Version is handled in the base class, as is name, the addPurlFacts is only used for special coordinate parts which are not handled in the standard way, like name which is according to Onur the namespace and in Maven, where getName returns groupId:artifactId